opslevel/datasource_opslevel_property_definition.go

```diff
@@ -0,0 +1,49 @@
+package opslevel
+
+import (
+	"github.com/opslevel/opslevel-go/v2023"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func datasourcePropertyDefinition() *schema.Resource {
+	return &schema.Resource{
+		Read: wrap(datasourcePropertyDefinitionRead),
+		Schema: map[string]*schema.Schema{
+			"identifier": {
+				Type:        schema.TypeString,
+				Description: "The id or alias of the property definition to find.",
+				Required:    true,
+			},
```

Comment:
```suggestion
			"identifier": {
				Type:        schema.TypeString,
				Description: "The id or alias of the property definition to find.",
				Required:    true,
			},
			"id": {
				Type:        schema.TypeString,
				Description: "The id of the property definition found.",
				Computed:    true,
			},
```
We need to ensure we can have a stable field in the schema for the ID since the customer could specify an alias for identifier